ni_measurementlink_service/_internal/stubs/ni/measurementlink/measurement/v1/measurement_service_pb2.pyi

```diff
@@ -45,17 +45,40 @@ class GetMetadataResponse(google.protobuf.message.Message):
     def measurement_signature(self) -> global___MeasurementSignature:
         """Required. Specifies the signature of the measurement."""
     @property
-    def user_interface_details(self) -> google.protobuf.internal.containers.RepeatedCompositeFieldContainer[global___UserInterfaceDetails]:
+    def user_interface_details(
```

Comment:
Did you run Black on the codegen? I thought we stopped doing that. Please update the excludes in pyproject.toml so this doesn't happen.

Reply:
I did run it. Will take a look at the pyproject.toml

Reply:
I don't think is there. Do you want me to add this file? Here? 
extend-exclude ='((.+_pb2_grpc)|(.+_pb2))+(.py)$' # Regex to files with _pb2_grpc.py and _pb2.py Auto Generated Code are excluded